js_modules/dagster-ui/packages/ui-core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -256,6 +268,13 @@ export const LaunchAssetExecutionButton: React.FC<{
                   onClick={(e) => onClick(option.assetKeys, e)}
                 />
               ))}
+              {!liveDataForStale && inScope.length && 'all' in scope ? (
```

Comment:
Only if not liveDataForStale? I think we should get rid of the other implementation so that there aren't two versions of this action on different pages in the app. I feel like people will forget to test one of the two.

I think it'd be fine to conditionally show this one if the features were the same, but this new one lets you uncheck assets and the old one does not.

Reply:
I'm wary of making too many changes at once but lets see what @braunjj  thinks